apps/web/src/pages/workflow/components/edit-modal/index.tsx

```diff
@@ -1 +1,59 @@
-// 添加/编辑弹窗组件
+import React from 'react';
+import { useMemoizedFn } from 'ahooks';
+import cls from 'classnames';
+import { useForm, Controller, type SubmitHandler } from 'react-hook-form';
+import { useI18n } from '@milesight/shared/src/hooks';
+import { Modal, toast, type ModalProps } from '@milesight/shared/src/components';
+import useEditFormItems, { type FormDataProps } from './hook/useEditFormItems';
+
+interface Props extends Omit<ModalProps, 'onOk'> {
+    /** error callback */
+    onError?: (err: any) => void;
+
+    /** success callback */
+    onSuccess?: (params: FormDataProps) => void;
+
+    /** table-row item */
+    data: FormDataProps;
```

Comment:
The component needs to support adding scenarios, so `data` should be an optional prop.